packages/sdk-client/src/sdk-middleware-auth/base-auth-flow.ts

```diff
@@ -92,12 +92,18 @@ async function executeRequest({
     }
 
     // Handle error response
-    let parsed
-    const text: any = await _res.text()
-    try {
-      parsed = JSON.parse(text)
-    } catch (error) {
-      /* noop */
+    let parsed: any;
+    let text: string;
+    if (!_res.ok) {
```

Comment:
We already made a check for `_res.ok` on line 59, I don't think this check here is necessary.